tequilapi/endpoints/proposals_test.go

```diff
@@ -89,7 +89,52 @@ func TestProposalsEndpointListByNodeId(t *testing.T) {
         }`,
 		resp.Body.String(),
 	)
-	assert.Equal(t, "0xProviderId", mockProposalProvider.recordedProviderId)
+	assert.Equal(t, "0xProviderId", mockProposalProvider.recordedProviderID)
+}
+
+func TestProposalsEndpointAcceptsAccessPolicyParams(t *testing.T) {
+	mockProposalProvider := &mockProposalProvider{
+		proposals: []market.ServiceProposal{serviceProposals[0]},
+	}
+
+	req, err := http.NewRequest(
+		http.MethodGet,
+		"/irrelevant",
+		nil,
+	)
+	assert.Nil(t, err)
+
+	query := req.URL.Query()
+	query.Set("accessPolicyId", "accessPolicyId")
+	query.Set("accessPolicySource", "accessPolicySource")
+	req.URL.RawQuery = query.Encode()
+
+	resp := httptest.NewRecorder()
+	handlerFunc := NewProposalsEndpoint(mockProposalProvider, &mysteriumMorqaFake{}).List
+	handlerFunc(resp, req, nil)
+
+	assert.JSONEq(
+		t,
+		`{
+            "proposals": [
+                {
+                    "id": 1,
+                    "providerId": "0xProviderId",
+                    "serviceType": "testprotocol",
+                    "serviceDefinition": {
+                        "locationOriginate": {
+                            "asn": 123,
+                            "country": "Lithuania",
+                            "city": "Vilnius"
+                        }
+                    }
+                }
+            ]
+        }`,
+		resp.Body.String(),
+	)
+	assert.Equal(t, "accessPolicySource", mockProposalProvider.recordedAccessPolicySource)
+	assert.Equal(t, "accessPolicyId", mockProposalProvider.recordedAccessPolicyID)
```

Comment:
40+ lines for asserting that our method passed the values, that's interesting :D